test/e2e/page-objects/pages/dialog/select-network.ts

```diff
@@ -86,6 +86,7 @@ class SelectNetwork {
     console.log(`Click ${networkName}`);
     const networkNameItem = `[data-testid="${networkName}"]`;
     await this.driver.clickElementAndWaitToDisappear(networkNameItem);
+    await this.driver.assertElementNotPresent('.loading-overlay');
```

Comment:
this step was in the spec, after switching networks, to wait until the loading spinner disappeared. Adding this step in the class level now, allow us to prevent a click being overlayed by the spinner, on any spec using this class.